content/docs/1.4.0/deploy/important-notes/index.md

```diff
@@ -59,3 +59,6 @@ The CSI snapshot CRDs `v1beta1` version is being deprecated and replaced by `v1`
 please follow the instruction [Enable CSI Snapshot Support](../../snapshots-and-backups/csi-snapshot-support/enable-csi-snapshot-support) to update CSI snapshot CRDs and CSI snapshot controller.
 If you have manifests or scripts that are still using `v1beta1` version, consider upgrading them to use `v1` as well.
 
+### Add storage class parameter `mkfsParams` and deprecate custom `mkfs.ext4` parameter
```

Comment:
Make `Custom mkfs.ext4 parameters`  to refer to the corresponding page.

```suggestion
### Add storage class parameter `mkfsParams` and `Custom mkfs.ext4 parameters` setting is deprecated
```

Reply:
I've fixed the title description and linked to the corresponding page.